### ISSUE TITLE
FR: Numbers for quicker result selection

### ISSUE THREAD

**Issue by chrisgrieser** [tadashi-aikawa/obsidian-another-quick-switcher#47]
some apps offer shortcuts to select specific search results, to save one from repeatedly clicking arrow-down or cmd/ctrl+j to get to a certain search result. Maybe something in that regard could also be a nice feature for AQS?
![image](https://user-images.githubusercontent.com/73286100/170961951-ae87e150-6c54-4568-af63-a5e087553dee.png)

**Comment by tadashi-aikawa** [comment]
Does it satisfy your requirements by changing from

```html
<span class="another-quick-switcher__item__hot-key-guide">[2]</span>
```

to

```html
<span class="another-quick-switcher__item__hot-key-guide">2</span>
```
 
?

**Comment by chrisgrieser** [comment]
sure, that would also work!

You can also simply add these to the `styles.css`, which would result in the same `[2]`, but woudl be more easier for theme devs to style
```css
.another-quick-switcher__item__hot-key-guide::before {
	content: "[";
}
.another-quick-switcher__item__hot-key-guide::after {
	content: "]";
}
```

**Comment by tadashi-aikawa** [comment]
@chrisgrieser 
Released in `v4.13.0-beta2` 🚀

**Comment by chrisgrieser** [comment]
Thanks! Now I could easily style the hotkey numbers the same way I style other hotkey information in my theme 😌 
<img width="1037" alt="Pasted image 2022-06-08 12 32 53" src="https://user-images.githubusercontent.com/73286100/172595737-c5202373-0a5f-4f47-91a8-db76556c087f.png">

Can't think of anything else to improve here 🙂

**Comment by tadashi-aikawa** [comment]
Released [v4.14.0](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/releases/tag/4.14.0) 🚀
